src/imageio.cs

```diff
@@ -2204,6 +2212,10 @@ interface CGImageDestinationOptions {
 		[Mac (11,0), iOS (14,0), TV (14,0), Watch (7,0)]
 		[Export ("WebPDictionary")]
 		NSDictionary WebPDictionary { get; set; }
+
+		[Mac (11,0), iOS (14,2), TV (14,2),  Watch (7,1)]
+		[Export ("TgaDictionary")]
+		NSDictionary TgaDictionary { get; set; }
```

Comment:
No Strong dict?

Reply:
That would be a nested strong dictionary heh, doable but a little out of the scope of this PR considering all the other docs already exposed as is in this particular dictionary